src/JuMP.jl

```diff
@@ -493,6 +515,10 @@ function addConstraint(m::Model, c::LinearConstraint)
     end
     return ConstraintRef{LinearConstraint}(m,length(m.linconstr))
 end
+addConstraint(m::Model, c::Array{LinearConstraint}) =
+    error("Vectorized constraint added without elementwise comparisons. Try using one of (.<=,.>=,.==).")
```

Comment:
I'd reword this as:
"The operators <=, >=, and == can only be used to specify scalar constraints. If you are trying to add a vectorized constraint, use the element-wise dot comparison operators (.<=, .>=, or .==) instead".